### PR TITLE
Fixes #23579 - Use a bundler group for assets

### PR DESCRIPTION
**Pull request by ekohl**

### CODE REVIEW

**Review by ohadlevy — DISMISSED** [review]

**Comment by ohadlevy** [comment]
any reason to wait for tests?

**Comment by ekohl** [comment]
Don't think there is but then again, I've been bitten by the "this can't possibly break" feeling before :)

**Comment by ohadlevy** [comment]
@ekohl looks like we need a change in infra to include the assets group..

**Comment by ohadlevy** [comment]
@ekohl any idea what needs to be fixed?

**Comment by ekohl** [comment]
@ohadlevy not at the moment. Going to have a look later but too many other issues right now and the meetup.

**Comment by ehelms** [comment]
You may need to add an explicit require here (https://github.com/theforeman/foreman/blob/develop/config/application.rb#L37) since this is not a common Rails group.

File: config/application.rb [hunk 1]
```diff
@@ -36,7 +36,7 @@ class Foreman::Consoletie < Rails::Railtie
     end
     Bundler.require(*Rails.groups)
     if SETTINGS[:unattended]
-      %w[ec2 fog gce libvirt openstack ovirt rackspace vmware].each do |group|
+      %w[assets ec2 fog gce libvirt openstack ovirt rackspace vmware].each do |group|
```
Comment (tbrisker) [inline config/application.rb]: We need the assets even if `SETTINGS[:unattended]==false`

**Comment by ekohl** [comment]
@tbrisker more like this?

**Review by tbrisker — APPROVED** [review]

**Review by ehelms — APPROVED** [review]

**Comment by ares** [comment]
Thanks for merging @ehelms, please don't forget to set the release version in redmine issue afterwards. Setting to 1.19

**Pull request merged**